README.md

```diff
@@ -24,3 +24,19 @@ Références :
 - [construire l'url](https://developer.matomo.org/guides/reporting-api-tutorial#build-the-url)
 - [appeler la méthode](Live.getLastVisitsDetails)
 - [l'idSite à utiliser](https://developer.matomo.org/api-reference/reporting-api)
+
+La procédure d'installation pour convertir un notebook en script (grace à `jupyter nbconvert`)
+```
+sudo apt-get install python3
+sudo apt-get install ipython
```

Comment:
Do you need both `jupyter` and `ipython`? On pip at least they’re supposed to be equivalent (jupyter is the new ipython).